plugin/EstateList.php

```diff
@@ -656,6 +661,89 @@ public function getEstateIds(): array
 		return array_column($this->_records, 'id');
 	}
 
+	/**
+	 * @param array $parkingArray
+	 * @param string $language
+	 * @param string $locate
+	 * @return array
+	 */
+	public function formatParkingLot(array $parkingArray, string $language, string $locale = 'de_DE'): array
+	{
+		$messages = [];
+		foreach ($parkingArray as $key => $parking) {
+			if (!$parking['Count']) {
+				continue;
+			}
+			switch ($language) {
+				case 'ENG':
+					$element = sprintf("%d %s at %s", number_format_i18n($parking['Count']), $this->getParkingName($key, $parking['Count']), $this->formatPrice($parking['Price'], $language, $locale));
```

Comment:
This part belongs into the template. Also, user-defined translations are possible in templates, so that no switch/case for the language is needed.